src/steganography.js

```diff
@@ -0,0 +1,136 @@
+// Loads input image to the Canvas for encoding or decoding
+function loadImage(e) {
+  let reader = new FileReader();
+  reader.onload = (event) => {
+    let dataUrl = event.target.result;
+    let img = new Image();
+    img.onload = () => {
+      let ctx = document.getElementById('canvas').getContext('2d');
+      ctx.canvas.width = img.width;
+      ctx.canvas.height = img.height;
+      ctx.drawImage(img, 0, 0);
+    }
+    img.src = dataUrl;
```

Comment:
You could maybe check if a non-image file is loaded. Feel free to open an issue and someone else might be interested to work on this.

Reply:
Thanks for the suggestion 
I'll look into it and merge it after
Thanks for the review!

Reply:
That's great! But it's okay and much better if you open an issue and let someone else work on this. They'd get an opportunity to understand the code and you can guide them if they've doubts.